app/views/sequence/new.html.erb

```diff
@@ -1,10 +1,42 @@
 <script src="https://cdn.jsdelivr.net/npm/vue@2.5.21/dist/vue.js"></script>
 <script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/vis/4.21.0/vis.min.js"></script>
 <link type="text/css" src="https://cdnjs.cloudflare.com/ajax/libs/vis/4.21.0/vis.min.css" />
+<link type="text/css" src="/assets/jsoneditor.min.css" />
 <script type="text/javascript" src="/assets/event-stream.js"></script>
 <script type="text/javascript" src="/assets/event-visualization.js"></script>
 <script type="text/javascript" src="/assets/event-creation-app.js"></script>
+<script type="text/javascript" src="/assets/jsoneditor.min.js"></script>
 
+<script>
+    // create the editor
+    function startJsonEditor(onChange) {
+      var container = document.getElementById("jsoneditor");
+      var options = {
+        onChange: () => onChange()
+      };
+      var editor = new JSONEditor(container, options);
+
+      // set json
+      var json = {
+        'merge_mode': 'merge'
+      };
```

Comment:
There's a default in the UI... I want to go further and put in config templates for each type.